addons/mail/static/src/components/attachment_delete_confirm/attachment_delete_confirm.js

```diff
@@ -2,16 +2,13 @@
 
 import { registerMessagingComponent } from '@mail/utils/messaging_component';
 import { useComponentToModel } from '@mail/component_hooks/use_component_to_model/use_component_to_model';
+import { escape } from '@web/core/utils/strings';
 
 const { Component } = owl;
 
 export class AttachmentDeleteConfirm extends Component {
-
-    /**
-     * @override
-     */
     setup() {
-        super.setup();
+        this.dialogAPI = {};
```

Comment:
The whole diff in the file, also not this line.

Reply:
yep sorry, I don't know what I did with this file